homeassistant/components/isy994/__init__.py

```diff
@@ -212,7 +215,13 @@ async def async_setup_entry(
 
     _categorize_nodes(hass_isy_data, isy.nodes, ignore_identifier, sensor_identifier)
     _categorize_programs(hass_isy_data, isy.programs)
+    # Categorize variables call to be removed with variable sensors in 2023.5.0
     _categorize_variables(hass_isy_data, isy.variables, variable_identifier)
+    # Gather ISY Variables to be added. Identifier used to enable by default.
+    for vtype, vname, vid in isy.variables.children:
+        hass_isy_data[ISY994_VARIABLES][Platform.NUMBER].append(
+            (isy.variables[vtype][vid], variable_identifier in vname)
+        )
```

Comment:
```suggestion
    numbers = hass_isy_data[ISY994_VARIABLES][Platform.NUMBER]
    for vtype, vname, vid in isy.variables.children:
        numbers.append(
            (isy.variables[vtype][vid], variable_identifier in vname)
        )
```

MINOR: I'd probably write it as above to avoid the dict lookups ever loop